package.json

```
@@ -9,34 +9,46 @@
   "main": "index.js",
   "license": "MIT",
   "scripts": {
-    "start": "concurrently \"yarn run start-js\" \"yarn run start-jsx\"",
+    "start": "npm-run-all --serial clean build-jsx --parallel start-jsx start-js",
     "start-js": "parcel src/js/index.html",
-    "start-jsx": "parcel watch src/jsx/index.jsx.ts --no-hmr --no-source-maps",
-    "build": "yarn run build-js && yarn run build-jsx",
+    "start-jsx": "parcel watch src/jsx/index.ts --no-hmr --no-source-maps",
+    "debug": "npm-run-all --parallel debug-jsx debug-js",
+    "debug-js": "parcel --log-level 4 src/js/index.html",
+    "debug-jsx": "parcel watch src/jsx/index.ts --no-hmr --no-source-maps --log-level 4",
+    "build": "npm-run-all clean build-js build-jsx",
```

```suggestion
    "build": "npm-run-all --serial clean build-jsx build-js",
```

This seems to fix the error I was experiencing with it not finding `./dist/index.js` and causing the build to break.